### PR TITLE
Rename "Firefox Lockbox" to "Firefox Lockwise" (Fixes #7242)

### LINKED ISSUE #7242: Rename "Firefox Lockbox" to "Firefox Lockwise" after May 28
## Description

Lockbox is getting a new name on 28 May, 2019 in time for the trailhead release. We'll need to find any pages where we refer to it by name and update them to Lockwise. We also need to update any links from lockbox.firefox.com to lockwise.firefox.com.

https://www.mozilla.org/firefox/accounts/ (the redesign (see #7132) already changes to Lockwise but most locales will still see the old page for a while until they can translate the new one)
https://www.mozilla.org/firefox/64.0/whatsnew/
https://www.mozilla.org/firefox/66.0/whatsnew/
https://www.mozilla.org/firefox/67.0/whatsnew/
https://www.mozilla.org/en-GB/firefox/new/ (SEO experiment)

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] All mentions of Lockbox are updated to Lockwise
- [ ] All links to lockbox.firefox.com are updated to lockwise.firefox.com

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] This is an example of a risk. Use a checklist!
- [ ] This is an example of a risk.

#### Tasks
- [ ] This is an example of a task. Use a checklist!
- [ ] This is an example of a task.

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Scoured bedrock for all instances of 'lockbox' 
_(reviewer: apologies for the dizzying amount of files 😅)_

## Issue / Bugzilla link
#7242 

## Testing
- Make sure that any instance of 'lockwise' inside a `{% trans %}` block or wrapped for l10n is properly staged. Using tag `lockwise_update_201906`
- All images containing the word 'lockbox' are renamed.

### CODE REVIEW

**Comment by amychurchwell** [comment]
Labeling as a size M review due to the amount of files touched.

**Comment by stephaniehobson** [comment]
Needs a rebase.

File: bedrock/foundation/templates/foundation/annualreport/2017/index.html [hunk 1]
```diff
@@ -301,15 +301,24 @@ <h3>{{ _('Continuously building our core to fuel growth') }}</h3>
...
-            {% trans vpn='https://www.engadget.com/2018/10/23/mozilla-firefox-protonvpn-test/',
-                     lockbox='https://testpilot.firefox.com/experiments/firefox-lockbox/'
-            %}
-              The overhaul of our core browser has given Mozilla Corporation the ability to begin the exploration
-              of new user engagement and revenue models derived from Firefox to meet these ambitious goals.
-              Specifically in 2018 we began testing in Firefox a new paid virtual private network
-              (<a href="{{ vpn }}">VPN</a>) service and a password manager called <a href="{{ lockbox }}">Lockbox</a>
-              which is currently being offered for free. Over time, with deep engagement with our users, we expect
-              to offer premium solutions.
+              {% trans vpn='https://www.engadget.com/2018/10/23/mozilla-firefox-protonvpn-test/',
```
Comment (stephaniehobson) [inline bedrock/foundation/templates/foundation/annualreport/2017/index.html]: I suspect that we can't nest if/else logic inside of trans statements without causing new strings to be created. Possibly all the code inside the trans statements ends up in the extracted strings?

You had the string extraction training and I didn't so you might be able to check this. I'm not sure how to verify.

To be on the safe side I would probably re-structure this to be more like:
```
{% set vpn='https://www.engadget.com/2018/10/23/mozilla-firefox-protonvpn-test/' %}

{% set vpn='https://www.engadget.com/2018/10/23/mozilla-firefox-protonvpn-test/' %}
{% set lockwise='https://testpilot.firefox.com/experiments/firefox-lockbox/' %}
{% set lockbox=lockwise %}
{% if l10n_has_tag('lockwise_update_201906') %}
{% trans vpn=vpn, lockwise=lockwise %} ... {% endtrans %}
{% else %}
{% trans vpn=vpn, lockbox=lockbox %} ... {% endtrans %}
{% endif %}

```

File: bedrock/foundation/templates/foundation/annualreport/2017/index.html [hunk 2]
```diff
@@ -301,15 +301,24 @@ <h3>{{ _('Continuously building our core to fuel growth') }}</h3>
...
+                of new user engagement and revenue models derived from Firefox to meet these ambitious goals.
+                Specifically in 2018 we began testing in Firefox a new paid virtual private network
+                (<a href="{{ vpn }}">VPN</a>) service and a password manager called <a href="{{ lockwise }}">Lockwise</a>
+                which is currently being offered for free. Over time, with deep engagement with our users, we expect
+                to offer premium solutions.
+              {% else %}
+                The overhaul of our core browser has given Mozilla Corporation the ability to begin the exploration
+                of new user engagement and revenue models derived from Firefox to meet these ambitious goals.
+                Specifically in 2018 we began testing in Firefox a new paid virtual private network
+                (<a href="{{ vpn }}">VPN</a>) service and a password manager called <a href="{{ lockwise }}">Lockbox</a>
```
Comment (stephaniehobson) [inline bedrock/foundation/templates/foundation/annualreport/2017/index.html]: I'm worried we can't change the variable name in a string without it being treated as a new string. Any way to check? Or maybe just  change it to play it safe.

File: media/css/firefox/accounts-2018.scss [hunk 3]
```diff
@@ -70,7 +70,7 @@ $image-path: '/media/protocol/img';
 // Modal, custom styles
 
 .mobile-notes,
-.mobile-lockbox,
+.mobile-lockwise,
```
Comment (stephaniehobson) [inline media/css/firefox/accounts-2018.scss]: +1 for updating the CSS 🔥

File: bedrock/firefox/templates/firefox/accounts-2018.html [hunk 4]
```diff
@@ -78,22 +78,39 @@ <h2 id="subtitle">{{ _('Own your life on every device with a Firefox Account.')
...
+    {% set lockwise_ga_title = 'Get the Lockbox App' %}
+  {% endif %}
+
   {% call feature_card(
-    title=_('Remember your passwords in Lockbox'),
+    title=lockwise_title,
     image_url='firefox/accounts/fxa-lockbox.png',
     include_highres_image=True,
     aspect_ratio='mzp-has-aspect-1-1',
     class='mzp-l-card-feature-left-half mzp-t-firefox lockbox-card qr-modal',
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/accounts-2018.html]: /s/lockbox-card/lockwise-card

File: bedrock/firefox/templates/firefox/whatsnew/fx64/whatsnew-fx64-base.html [hunk 5]
```diff
@@ -88,11 +88,11 @@ <h3>{{ notes_head }}</h3>
       <p><a class="mzp-c-button wn64-benefit-link" data-app="notes" href="https://play.google.com/store/apps/details?id=org.mozilla.testpilot.notes">{{ notes_link }}</a></p>
     </div>
 
-    <div class="wn64-benefit wn64-lockbox">
-      {{ high_res_img('firefox/whatsnew_64/fxa-lockbox.png', {'alt': '', 'width': '590', 'height': '590'}) }}
-      <h3>{{ lockbox_head }}</h3>
-      <p>{{ lockbox_desc }}</p>
-      <p><a class="mzp-c-button wn64-benefit-link" data-app="lockbox" href="https://app.adjust.com/orzhlh0">{{ lockbox_link }}</a></p>
+    <div class="wn64-benefit wn64-lockwise">
+      {{ high_res_img('firefox/whatsnew_64/fxa-lockwise.png', {'alt': '', 'width': '590', 'height': '590'}) }}
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/whatsnew/fx64/whatsnew-fx64-base.html]: The low res version of this file was renamed but the high-res on needs the rename too.

File: bedrock/firefox/templates/firefox/accounts-2018.html [hunk 6]
```diff
@@ -78,22 +78,39 @@ <h2 id="subtitle">{{ _('Own your life on every device with a Firefox Account.')
...
+    {% set lockwise_ga_title = 'Get the Lockwise App' %}
+  {% else %}
+    {% set lockwise_title = _('Remember your passwords in Lockbox') %}
+    {% set lockwise_ga_title = 'Get the Lockbox App' %}
+  {% endif %}
+
   {% call feature_card(
-    title=_('Remember your passwords in Lockbox'),
+    title=lockwise_title,
     image_url='firefox/accounts/fxa-lockbox.png',
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/accounts-2018.html]: Needs the updated filename.

File: bedrock/foundation/templates/foundation/annualreport/2017/index.html (hunk 1 above)
Comment (stephaniehobson) [reply bedrock/foundation/templates/foundation/annualreport/2017/index.html]: I think the failing test suggests we just can't nest if/else logic in trans statements.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.0.5.html [hunk 7]
```diff
@@ -67,15 +67,21 @@ <h2>{{ _('Including Firefox Monitor, your watchful eye for data breaches') }}</h
...
-            {% trans send='https://send.firefox.com/', lockwise='https://lockwise.firefox.com/', monitor='https://monitor.firefox.com/' %}
-            Firefox products like <a href="{{ send }}">Send</a>, <a href="{{ lockwise }}">Lockwise</a> and <a href="{{ monitor }}">Monitor</a> help you get things done without invading your privacy.
-            {% endtrans %}
+            {% if l10n_has_tag('lockwise_update_201906') %}
+              {% trans send='https://send.firefox.com/', lockwise='https://lockwise.firefox.com/', monitor='https://monitor.firefox.com/' %}
+              Firefox products like <a href="{{ send }}">Send</a>, <a href="{{ lockwise }}">Lockwise</a> and <a href="{{ monitor }}">Monitor</a> help you get things done without invading your privacy.
+              {% endtrans %}
+            {% else %}
+              {% trans send='https://send.firefox.com/', lockwise='https://lockwise.firefox.com/', monitor='https://monitor.firefox.com/' %}
+              Firefox products like <a href="{{ send }}">Send</a>, <a href="{{ lockwise }}">Lockbox</a> and <a href="{{ monitor }}">Monitor</a> help you get things done without invading your privacy.
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.0.5.html]: I think you will need to keep the variable name lockbox here too.

File: bedrock/firefox/templates/firefox/accounts-2018.html [hunk 8]
```diff
@@ -197,7 +214,13 @@ <h2 id="subtitle">{{ _('Own your life on every device with a Firefox Account.')
 
 <aside class="fxa-benefits mzp-u-modal-content">
   <p class="mobile-title mobile-pocket">{{ _('Scan the QR Code to download the Pocket App on your mobile device.') }}</p>
-  <p class="mobile-title mobile-lockbox">{{ _('Scan the QR Code to download the Lockbox App on your mobile device.') }}</p>
+  <p class="mobile-title mobile-lockwise">
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/accounts-2018.html]: Maybe as a result of the class that needs updating on  line 86?

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.0.5.html [hunk 9]
```diff
@@ -75,11 +75,11 @@ <h3>{{ _('Products that respect') }}</h3>
           <p>
             {% if l10n_has_tag('lockwise_update_201906') %}
               {% trans send='https://send.firefox.com/', lockbox='https://lockwise.firefox.com/', monitor='https://monitor.firefox.com/' %}
-              Firefox products like <a href="{{ send }}">Send</a>, <a href="{{ lockwise }}">Lockwise</a> and <a href="{{ monitor }}">Monitor</a> help you get things done without invading your privacy.
+              Firefox products like <a href="{{ send }}">Send</a>, <a href="{{ lockbox }}">Lockwise</a> and <a href="{{ monitor }}">Monitor</a> help you get things done without invading your privacy.
               {% endtrans %}
             {% else %}
               {% trans send='https://send.firefox.com/', lockwise='https://lockwise.firefox.com/', monitor='https://monitor.firefox.com/' %}
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.0.5.html]: The variable name here needs to be reverted to lockbox too.

**Pull request merged**